modules/persistent-storage-csi-cloning-overview.adoc

```diff
@@ -3,19 +3,27 @@
 // * storage/container_storage_interface/persistent-storage-csi-cloning.adoc
 
 [id="persistent-storage-csi-cloning-overview_{context}"]
-= Overview of CSI volume cloning
+= How CSI volume cloning works
```

Comment:
I know that the minimalism feedback hasn't been in favor of Overview of/Understanding in headings, but this heading looks odd to me. Especially because we don't use heading like this elsewhere in the OCP docs (that I'm aware of). I thought as a team we determined that we weren't going to worry about this, and could use "About xyz", etc.

Also, saying how something works isn't exactly interchangeable with an overview/general introduction of something. Just my 2 cents.